src/extension/extension.ts

```diff
@@ -338,22 +338,25 @@ export class RunmeExtension {
       ),
     )
 
+    await bootFile(context)
+
     if (isPlatformAuthEnabled()) {
       context.subscriptions.push(new StatefulAuthProvider(context, uriHandler))
-      const session = await getPlatformAuthSession(true)
-      if (session) {
-        const openDashboardStr = 'Open Dashboard'
-        const answer = await window.showInformationMessage(
-          'Logged into the Stateful Platform',
-          openDashboardStr,
-        )
-
-        if (answer === openDashboardStr) {
-          const dashboardUri = getRunmeAppUrl(['app'])
-          const uri = Uri.parse(dashboardUri)
-          env.openExternal(uri)
+      getPlatformAuthSession(true).then((session) => {
```

Comment:
I found an issue with this **window.InformationMessage**. The UI is being blocked until the dialog is closed.